lib/constants/wallet.js

```diff
@@ -19,3 +19,4 @@ export const SEND_TX_STATUS_SUCCESS = 'SEND_TX_STATUS_SUCCESS';
 export const SEND_TX_STATUS_FAILURE = 'SEND_TX_STATUS_FAILURE';
 export const SET_TEMPORARY_SECRETS = 'SET_TEMPORARY_SECRETS ';
 export const CLEAR_TEMPORARY_SECRETS = 'CLEAR_TEMPORARY_SECRETS ';
+export const CLEAR_ALL_WALLETS = 'CLEAR_ALL_WALLETS ';
```

Comment:
@bucko13 there is an extra space here, which is why `constants.CLEAR_ALL_WALLETS !== 'CLEAR_ALL_WALLETS'`

Reply:
Good catch. That makes a lot of sense! 😅 
Should the two above also be fixed then? That's probably where that snuck in since I just copy and pasted and then selected the matches to edit the text.